tests/test_kgain.py

```diff
@@ -0,0 +1,53 @@
+#A file to test the kgain conversion
+import os
+import numpy as np
+from corgidrp.mocks import create_default_headers
+import corgidrp.data as data
+import corgidrp.l2a_to_l2b as l2a_to_l2b
+import pytest
+
+def test_kgain():
+    """
+    test the KGain class and the calibration file and the unit conversion
```

Comment:
I think you should also test the case where the KGain class input is a file path where pri_hdr and ext_hdr is None.

Reply:
This is already done in line 24

Reply:
Oh, my mistake, I see that now.